planetscale/databases.go

```diff
@@ -36,13 +36,22 @@ type DeleteDatabaseRequest struct {
 	Database     string
 }
 
+// PromoteBranchRequest encapsulates the request for promoting a branch to
+// production.
+type PromoteBranchRequest struct {
+	Organization string `json:"-"`
+	Database     string `json:"-"`
```

Comment:
Any reason these two fields are omitted?

Reply:
@fatih Because they are used for creating the URL and not sent as part of the payload.